packages/client/lib/Chain.ts

```diff
@@ -101,6 +101,11 @@ export default class Chain implements ChainProvider, FeeProvider {
     return transaction
   }
 
+  /** @inheritdoc */
+  async call(to: Address | string, data: string): Promise<string> {
```

Comment:
This wouldn't have meaning in a non ethereum context? So it shouldn't be in the Chain interface, this place is reserved for functions that can be implemented cross chain.
You don't have to have something in the `Chain` to access it from providers. Just use `this.getMethod` in the provider directly